From 9093ffd69850b630622a3fc8dcf09c45c51ae2f9 Mon Sep 17 00:00:00 2001
From: John Ralls <jralls@ceridwen.us>
Date: Tue, 22 Mar 2022 12:09:09 -0700
Subject: [PATCH] Coretext: fix clang build failure.

Recent versions of clang notice that pango_core_text_fontset_load_font's
`key` local variable isn't used and errors out. Remove it.
---
 pango/pangocoretext-fontmap.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/pango/pangocoretext-fontmap.c b/pango/pangocoretext-fontmap.c
index 40e62eed..e090bfc4 100644
--- a/pango/pangocoretext-fontmap.c
+++ b/pango/pangocoretext-fontmap.c
@@ -1696,11 +1696,8 @@ static PangoFont *
 pango_core_text_fontset_load_font (PangoCoreTextFontset *ctfontset,
                                    CTFontDescriptorRef   ctdescriptor)
 {
-  PangoCoreTextFontsetKey *key;
   PangoCoreTextFont *font;
 
-  key = pango_core_text_fontset_get_key (ctfontset);
-
   /* For now, we will default the fallbacks to not have synthetic italic,
    * in the future this may be improved.
    */
-- 
2.32.0 (Apple Git-132)

